apps/calendar/jsdoc.json

```diff
@@ -5,11 +5,7 @@
   },
   "source": {
     "include": ["apps/calendar/README.md", "apps/calendar/js"],
-    "exclude": ["apps/calendar/js/ext",
```

Comment:
can you revert the changes to those files? otherwise it would *pollute* the history.